### ISSUE TITLE
Add translations for Tamil (ta)

### ISSUE THREAD

**Issue by kbdharun** [squidfunk/mkdocs-material#6740]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "ta",
  "direction": "ltr",
  "action.edit": "இந்த பக்கத்தை திருத்தவும்",
  "action.skip": "உள்ளடக்கத்திற்கு செல்க",
  "action.view": "இந்தப் பக்கத்தின் மூலத்தைப் பார்க்கவும்",
  "announce.dismiss": "இதை மீண்டும் காட்டாதே",
  "blog.archive": "காப்பகம்",
  "blog.categories": "வகைகள்",
  "blog.categories.in": "உள்ளே",
  "blog.continue": "தொடர்ந்து படிக்கவும்",
  "blog.draft": "வரைவு",
  "blog.index": "குறியீட்டுக்குத் திரும்பு",
  "blog.meta": "மெட்டாடேட்டா",
  "blog.references": "தொடர்புடைய இணைப்புகள்",
  "clipboard.copy": "கிளிப்போர்டுக்கு நகலெடுக்கவும்",
  "clipboard.copied": "கிளிப்போர்டுக்கு நகலெடுக்கப்பட்டது",
  "consent.accept": "ஏற்றுக்கொள்",
  "consent.manage": "அமைப்புகளை நிர்வகிக்கவும்",
  "consent.reject": "நிராகரி",
  "footer": "அடிக்குறிப்பு",
  "footer.next": "அடுத்தது",
  "footer.previous": "முந்தையது",
  "header": "தலைப்பு",
  "meta.comments": "கருத்துகள்",
  "meta.source": "ஆதாரம்",
  "nav": "வழிசெலுத்தல்",
  "readtime.one": "படிப்பதற்கு 1 நிமிடம்",
  "readtime.other": "படிப்பதற்கு # நிமிடம்",
  "rss.created": "RSS ஊட்டம்",
  "rss.updated": "புதுப்பிக்கப்பட்ட உள்ளடக்கத்தின் RSS ஊட்டம்",
  "search": "தேடு",
  "search.placeholder": "தேடு",
  "search.share": "பகிரவும்",
  "search.reset": "அழிக்கவும்",
  "search.result.initializer": "தேடலைத் துவக்குகிறது",
  "search.result.placeholder": "தேடலைத் தொடங்க தட்டச்சு செய்யவும்",
  "search.result.none": "பொருந்தக்கூடிய ஆவணங்கள் இல்லை",
  "search.result.one": "1 பொருந்தும் ஆவணம்",
  "search.result.other": "# பொருந்தும் ஆவணங்கள்",
  "search.result.more.one": "இந்தப் பக்கத்தில் மேலும் 1",
  "search.result.more.other": "இந்தப் பக்கத்தில் மேலும் #",
  "search.result.term.missing": "காணவில்லை",
  "select.language": "மொழியைத் தேர்ந்தெடுக்கவும்",
  "select.version": "பதிப்பைத் தேர்ந்தெடுக்கவும்",
  "source": "களஞ்சியத்திற்குச் செல்லவும்",
  "source.file.contributors": "பங்களிப்பாளர்கள்",
  "source.file.date.created": "உருவாக்கப்பட்டது",
  "source.file.date.updated": "கடைசி புதுப்பிப்பு",
  "tabs": "தாவல்கள்",
  "toc": "உள்ளடக்க அட்டவணை",
  "top": "மீண்டும் மேலே"
}[key] }}{% endmacro %}
```

### Country flag

flag_in

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to the best of my knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

---

I initially thought of creating a PR myself (got caught up in heavy work), thanks for the project; btw this is my commit email and name for your reference `K.B.Dharun Krishna <kbdharunkrishna@gmail.com>`.

**Comment by squidfunk** [comment]
Awesome, thanks! Added in 4fc33f68e.

**Comment by squidfunk** [comment]
Released as part of [9.5.8](https://squidfunk.github.io/mkdocs-material/changelog/#9.5.8).
